utils/build-script-impl

```diff
@@ -2329,7 +2330,15 @@ for host in "${ALL_HOSTS[@]}"; do
                         -DSWIFT_TENSORFLOW_TARGET_INCLUDE_DIR:PATH="${TENSORFLOW_TARGET_INCLUDE_DIR}"
                     )
 
-                    # Handle TensorFlow Swfit bindings file.
+                    # Propagate TensorFlow GPU support flag to CMake.
+                    if [[ "${ENABLE_TENSORFLOW_GPU}" ]] ; then
```

Comment:
who sets ${ENABLE_TENSORFLOW_GPU}?

Reply:
`build-script-impl` does this (capitalizes build flag names and sets them as environmental variables).

Reply:
build-script-impl does this (capitalizes build flag names and sets them as environment variables).

Reply:
I see. Thanks.